src/eei.cpp

```diff
@@ -177,8 +178,9 @@ inline int64_t maxCallGas(int64_t gas) {
       heraAssert(sp <= (1024 * stackItemSize), "EVM stack pointer out of bounds.");
       heraAssert(opcode >= 0x00 && opcode <= 0xff, "Invalid EVM instruction.");
 
-      auto it = evmInstructionNames.find(static_cast<uint8_t>(opcode));
-      string opName = (it != evmInstructionNames.end()) ? it->second : "UNKNOWN";
+      const char* opName = evmc_get_instruction_name_table()[static_cast<uint8_t>(opcode)];
+      if (opName == nullptr)
+        opName = "UNDEFINED";
```

Comment:
Is `undefined` used by the tracing format? The code above used `unknown` previously.

Reply:
I'm not sure tests depends on this, but the "undefined" name comes from https://github.com/ethereum/evmc/blob/master/include/evmc/evmc.h#L229.